<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Метролог клиент</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 8pt &quot;Bahnschrift&quot;;
	color: rgb(60, 66, 92);
	background-color: rgb(161, 204, 210);
}

QFrame {
	color: rgb(60, 66, 92);
	background-color: rgba(255, 255, 255, 70);
	border: none;
}

QLineEdit {
	border: 2px solid rgba(255, 255, 255, 70);
	border-radius: 3px;
	color: rgb(60, 66, 92);
	background-color: rgb(161, 204, 210);
}

QLineEdit:hover {
	border: 2px solid rgba(255, 255, 255, 70);
}

QLineEdit:focus {
	border: 2px solid rgb(255, 255, 255);
}

QPushButton {
	font: 12pt &quot;Bahnschrift&quot;;
	border-radius: 3px;
	color: rgb(60, 66, 92);
	background-color: rgb(255, 145, 132);
}

QPushButton:hover {
	font: 12pt &quot;Bahnschrift&quot;;
	border: 1px solid rgb(253, 252, 76);
	border-radius: 3px;
	color: rgb(253, 252, 76);
	background-color: rgb(255, 184, 172);
}

QPushButton:pressed {
	font: 12pt &quot;Bahnschrift&quot;;
	color: rgb(253, 252, 76);
	border: 2px solid rgb(60, 66, 92);
	background-color: rgba(56, 27, 85, 60);
}

QProgressBar:chunk {
	background-color: rgb(255, 145, 132);
    width: 5px;
    margin: 0.5px;
}

QTableView {
	font-size: 12px;
	border: none;
	gridline-color: rgba(255, 255, 255, 70);
	selection-color: rgb(253, 252, 76);
	selection-background-color: rgb(255, 145, 132);
}

QTableWiew:section {
	border: none;
	padding-bottom: 5px;
	padding-top: 5px;
}

QTableWiew:item {
	border-style: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>50</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>СИ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>ИО</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>ВО</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Копировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
